frontend/amundsen_application/static/js/features/ColumnList/ColumnDetailsPanel/index.spec.tsx

```diff
@@ -0,0 +1,278 @@
+
+      it('second button copies column link', () => {
+        const { props, wrapper } = setup();
+
+        wrapper.find('.btn-default').at(1).simulate('click');
+        const expected = getColumnLink(
+          props.columnDetails.tableParams,
+          props.columnDetails.name
+        );
+
+        expect(navigator.clipboard.writeText).toHaveBeenCalledWith(expected);
+      });
+    });
+
+    describe('renders column badges', () => {
+      it('should render badges', () => {
+        const { wrapper } = setup();
+
+        const actual = wrapper.find(BadgeList).length;
+        const expected = 1;
+
+        expect(actual).toEqual(expected);
+      });
+
+      it('should not render badges', () => {
+        const noBadgesColDetails = {
+          ...mockColumnDetails,
+          badges: [],
+        };
+        const { wrapper } = setup({ columnDetails: noBadgesColDetails });
+
+        const actual = wrapper.find(BadgeList).length;
+        const expected = 0;
+
+        expect(actual).toEqual(expected);
+      });
+    });
+
+    describe('renders a description', () => {
+      it('should render a description', () => {
+        const { wrapper } = setup();
+
+        const actual = wrapper.find(ColumnDescEditableText).length;
+        const expected = 1;
+
+        expect(actual).toEqual(expected);
+      });
+
+      it('should still render a description if only content is set', () => {
+        const withDescriptionColDetails = {
+          ...mockColumnDetails,
+          content: {
+            title: 'column_name',
+            description: 'description',
+            nestedLevel: 0,
+            hasStats: true,
+          },
+          editText: '',
+          editUrl: '',
+          isEditable: false,
+        };
+        const { wrapper } = setup({ columnDetails: withDescriptionColDetails });
+
+        const actual = wrapper.find(ColumnDescEditableText).length;
+        const expected = 1;
+
+        expect(actual).toEqual(expected);
+      });
+
+      it('should not render a description', () => {
+        const noDescriptionColDetails = {
+          ...mockColumnDetails,
+          content: {
+            title: 'column_name',
+            description: '',
+            nestedLevel: 0,
+            hasStats: true,
+          },
+          editText: '',
+          editUrl: '',
+          isEditable: false,
+        };
+        const { wrapper } = setup({ columnDetails: noDescriptionColDetails });
+
+        const actual = wrapper.find(ColumnDescEditableText).length;
+        const expected = 0;
+
+        expect(actual).toEqual(expected);
+      });
+    });
+
+    describe('renders column stats', () => {
+      it('should render stats', () => {
+        mockStats = mockColumnDetails.stats;
+        const { wrapper } = setup();
+
+        const actual = wrapper.find(ColumnStats).length;
+        const expected = 1;
+
+        expect(actual).toEqual(expected);
+      });
+
+      it('should not render stats', () => {
```

Comment:
Same here, if we don't set the context with the 'describe', it will be confusing to read the failures